examples/ChatQnA/deploy/k8s_manifest.md

```diff
@@ -0,0 +1,480 @@
+# Multi-node on-prem deployment with TGI on Xeon Scalable processors on a K8s cluster using Helm Charts
+<pod-name> in helm update
+
+This deployment section covers multi-node on-prem deployment of the ChatQnA example with OPEA comps to deploy using the TGI service. There are several slice-n-dice ways to enable RAG with vectordb and LLM models, but here we will be covering one option of doing it for convenience: we will be showcasing how to build an e2e chatQnA with Redis VectorDB and neural-chat-7b-v3-3 model, deployed on a Kubernetes cluster using Helm. For more information on how to setup a Xeon based Kubernetes cluster along with the development pre-requisites, follow the instructions here [Kubernetes Cluster and Development Environment](./k8s_getting_started.md#kubernetes-cluster-and-development-environment).
```

Comment:
Quoting or highlighting names where relevant, and/or splitting this to multiple paragraphs, could make this long text more readable.